checkpoint/client/cli/tx.go

```diff
@@ -45,6 +46,22 @@ func SendCheckpointTx(cdc *codec.Codec) *cobra.Command {
 		Short: "send checkpoint to tendermint and ethereum chain ",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cliCtx := context.NewCLIContext().WithCodec(cdc)
+			if viper.GetBool(FlagAutoConfigure) {
+				// fetch msg checkpoint
+				result, _, err := cliCtx.Query(fmt.Sprintf("custom/%s/%s", types.QuerierRoute, types.QueryNextCheckpoint))
+				if err != nil {
+					return err
+				}
+
+				// unmarsall the checkpoint msg
+				var newCheckpointMsg types.MsgCheckpoint
+				if err := json.Unmarshal(result, &newCheckpointMsg); err != nil {
+					return err
+				}
+
+				// broadcast this checkpoint
+				return helper.BroadcastMsgsWithCLI(cliCtx, []sdk.Msg{newCheckpointMsg})
```

Comment:
You can change the `msg.Proposer` just before the broadcast to change the signer.

Reply:
Non viable: Proposer gets checked in the handler.